prompt/select.ts

```diff
@@ -85,12 +113,31 @@ export class Select<S extends SelectSettings = SelectSettings>
     isSelected?: boolean,
   ): string {
     let line = this.settings.indent;
-    line += isSelected ? `${this.settings.listPointer} ` : "  ";
+
+    if (this.isCurrentlySearching()) {
+      line += this.settings.indent.repeat(item.indentLevel);
+      line += isSelected ? `${this.settings.listPointer} ` : "  ";
+    } else {
+      let pointer = "";
+      if (Select.itemIsBackButton(item)) {
+        // TODO: Make configurable
+        pointer = blue(`⮠`);
```

Comment:
I feel it's a nice touch that categories and the back button get their own pointers, but I assume they should be configurable, like the normal list pointer 😄

Reply:
Yeah configurable would be nice 👍